src/main/resources/migrations/00060_create_table_govukpay_events.sql

```diff
@@ -0,0 +1,20 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_table-govukpay_events
+CREATE TABLE govukpay_events (
+    id BIGSERIAL PRIMARY KEY,
+    mandate_id BIGINT,
+    payment_id BIGINT,
+    event_date TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'utc') NOT NULL,
+    resource_type VARCHAR(255),
```

Comment:
255 is probably a bit on the large size

Reply:
20?

Reply:
Maybe a little higher to give ourselves some breathing room. 32?

Reply:
ok

Reply:
updated